bindings/python/algorithm/expose-model.cpp

```diff
@@ -29,27 +31,49 @@ namespace pinocchio
       
       return bp::make_tuple(model,geom_model);
     }
-  
-    template<typename Scalar, int Options, template<typename,int> class JointCollectionTpl, typename ConfigVectorType>
-    bp::tuple
-    buildReducedModel(const ModelTpl<Scalar,Options,JointCollectionTpl> & model,
-                      const GeometryModel & geom_model,
-                      const std::vector<JointIndex> & list_of_joints_to_lock,
-                      const Eigen::MatrixBase<ConfigVectorType> & reference_configuration)
-    {
+
+    template <typename Scalar, int Options,
+              template <typename, int> class JointCollectionTpl,
+              typename ConfigVectorType>
+    bp::tuple buildReducedModel(
+        const ModelTpl<Scalar, Options, JointCollectionTpl> &model,
+        const std::vector<GeometryModel> &list_of_geom_models,
```

Comment:
You are changing the function argument. Please keep the former signature too as it will break he api otherwise

Reply:
I tried to keep the old signature.
I'm working on removing the formatting.
Hopefully it'll be clearer then.

One question though, I see you have a `cmake/.clang-format`, could I use that to format the C++ code?

Reply:
You can choose to only format your change, not everything normally. 
I will do a whole formating when switching to Pinocchio 3.x to avoid breaking the whole history.

Reply:
I see.

This function signature should be reverted now.